Assets/__Scripts/Map/Notes/BeatmapNote.cs

```diff
@@ -66,11 +67,39 @@ public class BeatmapNote : BeatmapObject
         return node;
     }
 
+    public Vector2 GetPosition()
+    {
+        if (_customData?.HasKey("_position") ?? false)
+        {
+            return _customData["_position"].ReadVector2();
+        }
+
+        float position = _lineIndex - 1.5f;
+        float layer = _lineLayer;
+
+        if (_lineIndex >= 1000)
+        {
+            position = (_lineIndex / 1000f) - 2.5f;
+        }
+        else if (_lineIndex <= -1000)
+        {
+            position = (_lineIndex / 1000f) - 0.5f;
+        }
+
+        if (_lineLayer >= 1000 || _lineLayer <= -1000)
+        {
+            layer = (_lineLayer / 1000f) - 1f;
+        }
+
+        return new Vector2(position, layer);
+    }
+
     protected override bool IsConflictingWithObjectAtSameTime(BeatmapObject other)
     {
         if (other is BeatmapNote note)
         {
-            return _lineIndex == note._lineIndex && _lineLayer == note._lineLayer;
+            // Only down to 1/4 spacing
+            return Vector2.Distance(note.GetPosition(), GetPosition()) < 0.25;
```

Comment:
I personally think 0.25 units is too much, since some Noodle Extensions maps (especially those made by cyan) have notes super close together, like so:

![super close notes](https://user-images.githubusercontent.com/30004286/90853251-658d9600-e32e-11ea-996f-e281db7622aa.png)